src/radical/entk/appman/appmanager.py

```diff
@@ -535,102 +505,58 @@ def resource_terminate(self):
         '''
-        **Purpose**: Setup RabbitMQ system on the client side. We instantiate
+        **Purpose**: Setup ZMQ system on the client side. We instantiate
         queue(s) 'pendingq-*' for communication between the enqueuer thread and
         the task manager process. We instantiate queue(s) 'completedq-*' for
         communication between the task manager and dequeuer thread. We
         instantiate queue 'sync-to-master' for communication from
         enqueuer/dequeuer/task_manager to the synchronizer thread. We
         instantiate queue 'sync-ack' for communication from synchronizer thread
         to enqueuer/dequeuer/task_manager.
-
-        Details: All queues are durable: Even if the RabbitMQ server goes down,
-        the queues are saved to disk and can be retrieved. This also means that
-        after an erroneous run the queues might still have unacknowledged
-        messages and will contain messages from that run. Hence, in every new
-        run, we first delete the queue and create a new one.
         '''
 
         try:
-            self._report.info('Setting up RabbitMQ system')
-            if self._mqs_setup:
+            sid = self._sid
+            self._report.info('Setting up ZMQ queues')
+
+            if self._zmq_info:
                 self._report.ok('>>n/a\n')
                 return
 
             self._report.ok('>>ok\n')
 
-            self._prof.prof('mqs_setup_start', uid=self._uid)
-            self._logger.debug('Setting up mq connection and channel')
-
-            mq_connection = pika.BlockingConnection(self._rmq_conn_params)
-
-            mq_channel = mq_connection.channel()
-
-            self._logger.debug('Connection and channel setup successful')
-            self._logger.debug('Setting up all exchanges and queues')
-
-            qs = ['%s-tmgr-to-sync' % self._sid,
-                  '%s-cb-to-sync'   % self._sid,
-                  '%s-sync-to-tmgr' % self._sid,
-                  '%s-sync-to-cb'   % self._sid]
-
-            for i in range(1, self._num_pending_qs + 1):
-                queue_name = '%s-pendingq-%s' % (self._sid, i)
-                self._pending_queue.append(queue_name)
-                qs.append(queue_name)
-
-            for i in range(1, self._num_completed_qs + 1):
-                queue_name = '%s-completedq-%s' % (self._sid, i)
-                self._completed_queue.append(queue_name)
-                qs.append(queue_name)
+            self._prof.prof('zmq_setup_start', uid=self._uid)
+            self._logger.debug('Setting up zmq queues')
 
-            for q in qs:
-                # Durable Qs will not be lost if rabbitmq server crashes
-                mq_channel.queue_declare(queue=q)
+            cfg = ru.Config(cfg={'channel'   : sid,
+                                 'uid'       : sid,
+                                 'path'      : sid,
+                                 'type'      : 'queue',
+                                 'log_lvl'   : 'DEBUG',
```

Comment:
should we read log level from a corresponding env variable?

Reply:
You are right - I removed the fixed setting, the env will kick in from [here](https://github.com/radical-cybertools/radical.utils/blob/devel/src/radical/utils/zmq/bridge.py#L46) where the `cfg` setting will be `None` now.